CONTRIBUTING.md

```diff
@@ -10,14 +10,36 @@ To discuss anything please [contact Cornelius](mailto:cschum@suse.de).
 
 ## Contribute code
 
-To contribute code please open pull requests.
+To contribute code fork the repository and open pull requests with your changes.
 
 Ensure that rspec and rubocop pass locally before sending your PR and always that you add new changes.
 
 If your changes include important new features or bug fixes please add them to the [Master (unreleased) section fo the CHANGELOG.md](https://github.com/openSUSE/trollolo/blob/master/CHANGELOG.md#master-unreleased)
 
+### Set up
+
+Clone git repo: 
+
+`git clone git@github.com:openSUSE/trollolo.git`
+
+Build the gem: 
+
+`gem build trollolo.gemspec`
+
+Install the gem: 
+
+`gem install trollolo-0.2.0.gem`
+
+To test and debug changes in Interactive Ruby Shell: 
+
+`irb -Ilib -rtrollolo`
```

Comment:
why do you need to do this?

Reply:
To test minor changes, this command is equivalent to `require 'trollolo'` and opening the irb. I thought it could be handy to debug changes by calling functions and logging required variables wherever necessary.

Reply:
but why do you need `irb` to test Trollolo changes? 😕

Reply:
yes, it's not necessary, I thought it could be useful  :stuck_out_tongue: , i'll remove this